### PR TITLE
chore(examples): update e-commerce demo

### PR DESCRIPTION
**Pull request by francoischalifour**
This updates the examples' webpack configuration to allow images and web manifests.

### CODE REVIEW

File: website/webpack.config.js [hunk 1]
```diff
@@ -39,24 +39,60 @@ module.exports = {
...
         new HTMLWebpackPlugin({
           template: path.join(__dirname, example, 'index.html'),
           filename: path.join(outputPath, example, 'index.html'),
           chunks: [example],
         })
-    )
-    .concat([
-      new CopyWebpackPlugin([
-        { from: path.join(__dirname, 'assets'), to: 'assets/' },
-        path.join(__dirname, '_redirects'),
```
Comment (samouss) [inline website/webpack.config.js]: It looks like we forgot this line inside the change. The redirects don't work anymore.

> https://deploy-preview-2531--react-instantsearch.netlify.com/

File: website/webpack.config.js [hunk 2]
```diff
@@ -39,24 +39,60 @@ module.exports = {
...
+              outputPath(_url, resourcePath, context) {
+                return path.relative(context, resourcePath);
+              },
+              name: '[name].[ext]',
+            },
+          },
+        ],
+      },
+      {
+        test: /\.webmanifest$/,
```
Comment (samouss) [inline website/webpack.config.js]: The images don't use the hash for the name, so what is the purpose of the loaders for the manifest? It doesn't work with a plain copy? Nothing seems to be dynamic inside the manifest.

Comment (francoischalifour) [reply website/webpack.config.js]: That was the first solution but we then changed that with @yannickcr.

I updated to use `CopyWebpackPlugin` which seems to be easier to use in this case.

File: website/webpack.config.js (hunk 1 above)
Comment (francoischalifour) [reply website/webpack.config.js]: Put back in 816ec028ef46b7d54040d51f602e1f82da1f39a1.

File: website/webpack.config.js [hunk 3]
```diff
@@ -88,6 +69,15 @@ module.exports = {
           chunks: [example],
         })
     ),
+    ...examples.map(
+      example =>
+        new CopyWebpackPlugin([
```
Comment (samouss) [inline website/webpack.config.js]: I don't think we have to call the plugin again, we can provide an array of paths as an argument.

Comment (francoischalifour) [reply website/webpack.config.js]: I tried that but it was a nightmare. Having a regex in the `from` and using it in the `to` doesn't work as expected.

Comment (samouss) [reply website/webpack.config.js]: Not sure to follow, we can't do this?

```js
[
  new CopyWebpackPlugin([
    ...examples.map(example => ({
      from: path.join(__dirname, example, "assets"),
      to: path.join(outputPath, example, "assets")
    })),
    {
      from: path.join(__dirname, "assets"),
      to: "assets/"
    },
    path.join(__dirname, "_redirects")
  ])
]
```

Comment (francoischalifour) [reply website/webpack.config.js]: Ah right! Yes, this works!

**Pull request merged**